tests/test_migrator.py

```diff
@@ -125,18 +129,9 @@ def test_foreign_key(tmpdir):
         table.primary_key('id')
         table.foreign_key('int', 'basic_id', 'basic')
 
-    with manager.migrator.create_table('related2') as table:
```

Comment:
It seems that peewee now names the indexes relative to the model it is in, in this lib's case, `FakeModel` so then it attempts to create the index with that model's name as "prefix" :/